lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/SherllinLesson7/Dog.java

```diff
@@ -0,0 +1,69 @@
+package com.codedifferently.lesson7.SherllinLesson7;
```

Comment:
nit: Package name should be lowercase (so too the folder name `SherllinLesson7`.